castle/cms/browser/controlpanel/templates/sessions.pt

```diff
@@ -25,6 +25,11 @@
     </div>
 
     <div id="content-core">
+      Session Inactivity Timeout:
```

Comment:
If we are going to add this configuration, it should be in the security control panel IMO.

Reply:
Seems this entire sessions panel relates to security, I just thought this would be the place since it is about session configuration.  Open to moving it I suppose.

Reply:
The session control panel doesn't have any configuration though. It's for inspecting active user sessions, not for configuration anything.

I imagine there are cases where we want a type of user to be able to inspect user session(site administrator) but not configure session duration(manager).

But, I'm open to allowing whatever everyone wants here. I just thought it looked weird there.

Reply:
I agree with Nathan that this setting belongs more in the security control panel. The Sessions panel is actually named Active Sessions when you're viewing it, which implies that it's meant to look at them (and yeah maybe end them). (And I'd propose renaming the panel so that it shows up as "Active Sessions" when you're looking at @@security-controlpanel)